src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { Header } from '../Header/Header';
```

Comment:
You don't have index.ts files in your folders and you need to write Header 2 times in imports. Please, add index files

Reply:
fixed